llvm/test/tools/sycl-post-link/emit_program_metadata.ll

```diff
@@ -12,14 +12,16 @@ attributes #2 = { "sycl-work-group-size"="4" }
 
 !0 = !{i32 1, i32 2, i32 4}
 !1 = !{i32 2, i32 4}
-!2 = !{i32 4}
+!2 = !{i64 2, i64 4}
+!3 = !{i32 4}
+!4 = !{i64 4}
 
 define weak_odr spir_kernel void @SpirKernel1(float %arg1) !reqd_work_group_size !0 {
   call void @foo(float %arg1)
   ret void
 }
 
-define weak_odr spir_kernel void @SpirKernel2(float %arg1) #0 {
+define weak_odr spir_kernel void @SpirKernel2(float %arg1) #0 !reqd_work_group_size !1 {
```

Comment:
I think we can completely remove kernels which had attributes, but not metadata. Since the only input for `sycl-post-link` now is metadata, then there is no need to effectively duplicate test cases: like `!1` and `!2` are effectively the same: why do we need to have two separate kernels with the same metadata?

Reply:
Removed these in latest patch.